pkgs/os-specific/linux/realmd/default.nix

```diff
@@ -0,0 +1,73 @@
+{ lib, stdenv
+, autoreconfHook
+, docbook_xml_dtd_412
+, docbook_xml_dtd_42
+, docbook_xml_dtd_43
+, docbook_xsl
+, fetchFromGitLab
+, glib
+, intltool
+, libkrb5
+, libtool
+, libxslt
+, openldap
+, pkg-config
+, polkit
+, systemd
+, xmlto
+}:
+# doc-related (--disable-doc): docbook_xml_dtd_412 docbook_xml_dtd_42 docbook_xml_dtd_43 docbook_xsl libxslt xmlto
+
+stdenv.mkDerivation rec {
+  pname = "realmd";
+  version = "0.17.0";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.freedesktop.org";
+    owner = pname;
+    repo = pname;
+    rev = version;
+    sha256 = "1c6q2a86kk2f1akzc36nh52hfwsmmc0mbp6ayyjxj4zsyk9zx5bf";
+  };
+
+  configureFlags = [
+    "--with-distro=NixOS"
+    "--with-systemd-unit-dir=${placeholder "out"}/lib/systemd/system"
+  ];
+
+  nativeBuildInputs = [
+    autoreconfHook
+    docbook_xml_dtd_412
+    docbook_xml_dtd_42
+    docbook_xml_dtd_43
+    docbook_xsl
+    glib
+    intltool
+    libkrb5
+    libtool
+    libxslt
+    openldap
+    pkg-config
+    polkit
+    systemd
+    xmlto
```

Comment:
```suggestion
    pkg-config

  buildInputs = [
    glib
    intltool
    libkrb5
    libxslt
    openldap
    polkit
    systemd
    xmlto
  ];
```

Reply:
Without `pkg-config` in `buildInputs`, it fails:
```
checking for XML::Parser... ok
checking for pkg-config... no
checking for locale.h... yes
checking for LC_MESSAGES... yes
checking for CFPreferencesCopyAppValue... no
checking for CFLocaleCopyCurrent... no
checking for libintl.h... yes
checking for ngettext in libc... yes
checking for dgettext in libc... yes
checking for bind_textdomain_codeset... yes
checking for msgfmt... (cached) /nix/store/mnfr685721nrrv1wb8pilbdnl2d57gm5-gettext-0.21/bin/msgfmt
checking for dcgettext... yes
checking if msgfmt accepts -c... yes
checking for gmsgfmt... (cached) /nix/store/mnfr685721nrrv1wb8pilbdnl2d57gm5-gettext-0.21/bin/msgfmt
checking for xgettext... (cached) /nix/store/mnfr685721nrrv1wb8pilbdnl2d57gm5-gettext-0.21/bin/xgettext
./configure: line 7662: syntax error near unexpected token `GLIB,'
./configure: line 7662: `PKG_CHECK_MODULES(GLIB,'
builder for '/nix/store/1fjdp5bckxc81x8s0vsynmrjb8p453n2-realmd-0.17.0.drv' failed with exit code 2
```

Applied your suggestion + re-added `pkg-config`.